.profile

```diff
@@ -141,6 +141,9 @@ export CKANEXT__DATAGOVCATALOG__ADD_PACKAGES_TRACKING_INFO=false
 # Render recent view using AJAX call to boost page loading speed
 export CKANEXT__DATAGOVTHEME__JS_RECENT_VIEW=true
 
+# Max number of resources to be allowed in a dataset to be harvested
+export CKANEXT__DATAJSON__MAX_RESOURCE_COUNT=1500
+
```

Comment:
Any reason we can't add this to the ckan.ini file? I think this would be the safer place; that way it will be setup by default. I don't see a reason to have a different number between local dev and cloud.gov deployment...

Reply:
Tried that before, moved newly created config into ckan.ini but it had no effect.
Just tried again with https://github.com/GSA/catalog.data.gov/commit/50e405594ab9c334cb8232137540f029560a3753, same result. No effect when set in ckan.ini.

Reply:
oops, looks like there is a typo in the last try. trying again.